2017/2--corruption-checksum/crystal/solution.cr

```diff
@@ -1,3 +1,12 @@
 def part1(spreadsheet : String)
-	spreadsheet.split("\n").map { |r| r.split(/\s+/).map { |c| c.to_i } }.map { |r| r.max - r.min }.sum
```

Comment:
> The go version would be easier to maintain I think

It always depend the way you code,
Developers can write unreadable code in almost every language 😉